packages/frontend/src/components/UserSettings/ProfilePanel/index.tsx

```diff
@@ -90,14 +88,14 @@ const ProfilePanel: FC = () => {
     }, [
         sendVerificationEmailError,
         data,
-        error,
+        updateUserError,
```

Comment:
you'll probably deal with this in the next PR? to remove useEffect and move it to onError callback right?

Reply:
done on this one!